linux_os/guide/services/sssd/sssd-ldap/sssd_ldap_configure_tls_reqcert/tests/correct_value_dropin.pass.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+. $SHARED/setup_config_files.sh
+setup_correct_sssd_config
+
+sed -i '/ldap_tls_reqcert/d' /etc/sssd/sssd.conf
+
+echo '[domain/default]' >> /etc/sssd/conf.d/cac.conf
+echo 'ldap_tls_reqcert = demand' >> /etc/sssd/conf.d/cac.conf
+systemctl enable sssd
```

Comment:
Should we include a header to ensure the sssd package is installed? Otherwise this command can fail where sssd is not installed by default.